### PR TITLE
Create 1.9 jobs

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Review by ericvn — APPROVED** [review]

File: prow/config/jobs/tools-1.9.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  resources: build
+  types:
+  - presubmit
+- command:
+  - entrypoint
+  - make
+  - -C
+  - perf_dashboard
+  - deploy
+  name: deploy-perf-dashboard
```
Comment (howardjohn) [inline prow/config/jobs/tools-1.9.yaml]: I think we only want this on master

File: prow/config/jobs/proxy-1.9.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+- command:
+  - ../test-infra/tools/automator/automator.sh
+  - --org=istio
+  - --repo=istio
+  - '--title=Automator: update proxy@$AUTOMATOR_SRC_BRANCH in $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH'
+  - --labels=auto-merge,release-notes-none
+  - --modifier=update_proxy_dep
+  - --token-path=/etc/github-token/oauth
+  - --cmd=bin/update_proxy.sh $AUTOMATOR_SHA
+  image: gcr.io/istio-testing/build-tools:master-2021-01-13T15-02-13
```
Comment (howardjohn) [inline prow/config/jobs/proxy-1.9.yaml]: failed to update?

File: prow/config/jobs/proxy-1.9.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  - ./prow/proxy-presubmit-release.sh
+  name: release-test
+  requirements:
+  - gcp
+  timeout: 4h0m0s
+  types:
+  - presubmit
+- command:
+  - ./prow/proxy-presubmit-centos-release.sh
+  image: gcr.io/istio-testing/build-tools-centos:master-2021-01-13T15-02-13
```
Comment (howardjohn) [inline prow/config/jobs/proxy-1.9.yaml]: wrong image

**Comment by jacob-delgado** [comment]
For https://github.com/istio/istio/issues/29490

**Pull request merged**